### DISCUSSION TITLE
Add ability to use online map tiles without downloading them. (This would make faster and easier to contribute to OSM, to add PoIs and Notes.

### DISCUSSION THREAD

**Discussion by d4f5409d** [osmandapp/OsmAnd#12493] [Q&A]
# 🚀 feature request

### Describe the solution you'd like

 If you have a solution in mind, please describe it.

### Describe alternatives you've considered

 Have you considered any alternative solutions or workarounds?

**Comment by vshcherb** [comment]
You probably mean to use offline maps (binary maps) cause it's already possible to use online tiles without downloading packages.

**Reply by d4f5409d** [reply]
No, for me it doesn't let me use it offline without downloading it. Is this a feature in 4.0?

**Reply by vshcherb** [reply]
You can switch plugin Online Maps, then select OsmAnd online tiles as map source
